data/mods/Xedra_Evolved/mapgen/overmap_terrain.json

```
@@ -159,5 +159,48 @@
     "looks_like": "standing_stones",
     "see_cost": 5,
     "mondensity": 2
+  },
+  {
+    "type": "overmap_terrain",
+    "id": [ "salamander_eruption_ground" ],
+    "name": "salamander eruption",
+    "sym": "E",
+    "color": "red_cyan",
+    "see_cost": 5,
+    "flags": [ "KNOWN_DOWN" ]
```

```suggestion
    "flags": [ "KNOWN_DOWN", "NO_ROTATE" ]
```
Mutables freely rotate stuff with vertical joins and bc the z0 root is placed separately there's no easy way to have the stack have the same rotation so I've just prevented it rotating these at all so the slopes line up, the start of the tunnels is a crossroad anyway so it doesn't really make much difference